Source/Core/VideoCommon/FreeLookCamera.h

```diff
@@ -36,50 +57,41 @@ class CameraController
 
   virtual void Reset() = 0;
 
-  virtual void DoState(PointerWrap& p) = 0;
+  void IncreaseFovX(float fov);
+  void IncreaseFovY(float fov);
+  float GetFovStepSize() const;
+
+  void ModifySpeed(float multiplier);
+  void ResetSpeed();
+  float GetSpeed() const;
+
+private:
+  float m_fov_x = 1.0f;
+  float m_fov_y = 1.0f;
+
+  float m_min_fov_multiplier = 0.025f;
```

Comment:
It seems odd to me that this isn't a constant; I'd expect `static constexpr float MIN_FOV_MULTIPLIER = 0.025f` instead.  (It also seems odd that this one is specified as a multiplier while the others aren't.)